packages/utils/tool-utils/src/snapshotNormalizer.ts

```diff
@@ -110,39 +111,54 @@ function getNormalizedBlobContent(blobContent: string, blobName: string): string
 export function getNormalizedSnapshot(snapshot: ITree, config?: ISnapshotNormalizerConfig): ITree {
     // Merge blobs to normalize in the config with runtime blobs to normalize. The contents of these blobs will be
     // parsed and deep sorted.
-    const blobsToNormalize = [ ...config?.blobsToNormalize ?? [] ];
     const normalizedEntries: ITreeEntry[] = [];
 
     for (const entry of snapshot.entries) {
-        switch (entry.type) {
```

Comment:
moved this to the below function the make the logic a bit easier, as they i could use return